applications/StructuralMechanicsApplication/custom_processes/postprocess_eigenvalues_process.cpp

```diff
@@ -145,28 +145,33 @@ struct VtkEigenOutputWrapper : public BaseEigenOutputWrapper
 
 } // helpers namespace
 
-PostprocessEigenvaluesProcess::PostprocessEigenvaluesProcess(ModelPart& rModelPart,
-                                                                Parameters OutputParameters)
-                                                                : mrModelPart(rModelPart),
-                                                                mOutputParameters(OutputParameters)
+PostprocessEigenvaluesProcess::PostprocessEigenvaluesProcess(
+    Model& rModel,
+    Parameters OutputParameters)
+    : mOutputParameters(OutputParameters)
 {
-    Parameters default_parameters(R"(
-        {
-            "result_file_name"              : "Structure",
-            "file_format"                   : "vtk",
-            "file_label"                    : "step",
-            "result_file_format_use_ascii"  : false,
-            "folder_name"                   : "EigenResults",
-            "save_output_files_in_folder"   : true,
-            "animation_steps"               : 20,
-            "label_type"                    : "frequency",
-            "list_of_result_variables"      : ["DISPLACEMENT"]
-        }  )"
-    );
-
-    mOutputParameters.RecursivelyValidateAndAssignDefaults(default_parameters);
+    mOutputParameters.RecursivelyValidateAndAssignDefaults(GetDefaultParameters());
+    mpModelPart = &(rModel.GetModelPart(mOutputParameters["model_part_name"].GetString()));
+    const std::string folder_name = mOutputParameters["folder_name"].GetString();
+    if (mOutputParameters["wipe_results_folder"].GetBool()) {
+        std::filesystem::remove_all(folder_name);
+    }
+    if (!std::filesystem::exists(folder_name)) {
+        std::filesystem::create_directories(folder_name);
+    }
+}
 
+PostprocessEigenvaluesProcess::PostprocessEigenvaluesProcess(
+    ModelPart& rModelPart,
+    Parameters OutputParameters)
+    : mpModelPart(&rModelPart)
+    , mOutputParameters(OutputParameters)
```

Comment:
why is this constructor still necessary?

I thought thats what you refactored? I.e. can it be removed?

Reply:
I left it in there because of backward compatibility. Adding to my comment that the position of the @KratosMultiphysics/technical-committee in this regard is to promote Model-parameters ones and allow alternative ones in case there is a technical need. 

Having said this, if you consider that we can remove it without any harm I'd remove it.

Reply:
as long as it works from python via project parameters I think we are ok

If someone complains we can still do sth abt it :)

Reply:
Very good. I'll remove it then.

Reply:
Done.